Sources/PointFree/Privacy.swift

```diff
@@ -0,0 +1,174 @@
+import Css
+import CssReset
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Prelude
+import Styleguide
+
+private let title = "Privacy Policy"
+
+let privacyResponse =
+  writeStatus(.ok)
+    >-> map(lower)
+    >>> respond(
+      view: privacyView,
+      layoutData: { currentUser in
+        SimplePageLayoutData(currentUser: currentUser, data: unit, title: title)
+    }
+)
+
+private let privacyView = View<Prelude.Unit> { _ in
+  gridRow([
+    gridColumn(sizes: [.mobile: 12], [
+      div(
+        [`class`([Class.padding([.mobile: [.all: 4]])])],
+        [h1([`class`([Class.h1])], [text(title)])]
+          <> privacyPolicy
+          <> [
+            p(
+              [`class`([Class.padding([.mobile: [.top: 2]])])],
+              ["This document was last updated on January 7, 2018."]
+            )
+        ]
+      )
+      ])
+    ])
+}
+
+private let privacyPolicy =
+  personalIdentificationInformation
+    <> nonPersonalIdentificationInformation
+    <> webBrowserCookies
+    <> howWeUseCollectedInformation
+    <> howWeProtectYourInformation
+    <> sharingYourPersonalInformation
+    <> complianceWithChildrensOnlinePrivacyProtectionAct
+    <> changesToThisPrivacyPolicy
+    <> contactingUs
+
+private let personalIdentificationInformation = [
+  h2(
+    [`class`([Class.h3, Class.padding([.mobile: [.top: 2]])])],
```

Comment:
we should use `Class.pf.type.title3` instead of `Class.h3` cause it sets color and line-heights in addition to font size